src/fmu_interfaces/CMakeLists.txt

```diff
@@ -0,0 +1,30 @@
+#++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++!
+#                          Futility Development Group                          !
+#                             All rights reserved.                             !
+#                                                                              !
+# Futility is a jointly-maintained, open-source project between the University !
+# of Michigan and Oak Ridge National Laboratory.  The copyright and license    !
+# can be found in LICENSE.txt in the head directory of this repository.        !
+#++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++!
+
+INCLUDE_DIRECTORIES(${CMAKE_CURRENT_BINARY_DIR})
+INCLUDE_DIRECTORIES(${CMAKE_CURRENT_SOURCE_DIR})
+
+TRIBITS_ADD_LIBRARY(FmuUtils
+    HEADERS
+      FMU.h
+      FMU2.h
+      fmi2FunctionTypes.h
+      fmi2Functions.h
+      fmi2TypesPlatform.h
+      FMU2_c_api.cpp
+      FMU2_c_api.h
+      FMU2_f_api.f90
+    SOURCES
+      FMU.cpp
+      FMU2.cpp
+)
+
+MESSAGE("-- Enabling Futility FMU Wrappers. CMAKE_DL_LIBS: " ${CMAKE_DL_LIBS})
```

Comment:
So normally we tend to do `MESSAGE(STATUS "Enabling Futility...` instead... having the `STATUS` keyword there actually prepends the `-- ` part that you added.  It's not really a big deal, but for consistency would be good to change

Reply:
fixed